src/blockchain/rpc.rs

```diff
@@ -456,3 +509,49 @@ crate::bdk_blockchain_tests! {
         RpcBlockchain::from_config(&config).unwrap()
     }
 }
+
+#[cfg(test)]
+#[cfg(feature = "test-rpc")]
+mod test {
+    use super::*;
+    use crate::blockchain::*;
+    use crate::testutils::blockchain_tests::TestClient;
+
+    use bitcoin::Network;
+    use bitcoincore_rpc::RpcApi;
+
+    #[test]
+    fn test_rpc_blockchain_factory() {
```

Comment:
It'd be nice to have a simple test like this for one of the stateless blockchain factories like `ElectrumBlockchain`.